swift/ql/test/library-tests/dataflow/dataflow/test.swift

```diff
@@ -692,4 +692,38 @@ func testArray() {
     var arr6 = [1,2,3]
     arr6.insert(source(), at: 2)
     sink(arg: arr6[0]) // $ flow=693
+
+    var arr7 = [source()]
+    sink(arg: arr7.randomElement()!) // $ flow=696
+
+    for entry in arr7
+    {
+        sink(arg: entry) // $ flow=696
+    }
```

Comment:
I doubt it will affect flow, but it might also be nice to have one test in the alternative form:
```
arr7.forEach { entry in
	sink(arg: entry) // $ flow=
}
```

Reply:
Ah, then I'll add that to the list of closure methods we plan to model!